Ecommerce/index.css

```diff
@@ -22,4 +22,9 @@ table {
         }
         th {
             background-color: #f2f2f2;
+        }
+

```

Comment:
The color property in the body selector should be indented to align with the body selector.